django/publicmapping/Dockerfile

```diff
@@ -2,6 +2,13 @@ FROM quay.io/azavea/django:1.11-python2.7-slim
 
 RUN mkdir -p /usr/src/app
 RUN mkdir -p /opt/sld
+
+RUN mkdir -p /opt/reports
+
+# Create reporter user that can write to reports directory
```

Comment:
This is to keep celery from complaining about being run as root, right? That might be worth a comment to explain because it's a bit unusual to see user permissions stuff inside a Dockerfile.

Reply:
I added some more comments just after line 8